### PR TITLE
[CPLP-1460] - input field validation in step 2

### PR DESCRIPTION
**Pull request by smartdeepp**

### CODE REVIEW

**Comment by jjeroch** [comment]
@nidhigarg-bmw functional wise the implementation looks good, but it would be great if you can do a code quality check - mainly if the way how the code was written is as expected :)

File: src/components/cax-responsibilities.tsx [hunk 1]
```diff
@@ -101,14 +109,29 @@ export const ResponsibilitiesCax = ({
 
   const validateEmailOnChange = (email) => {
     setEmail(email)
-    if (email === '')
-      setError({ email: 'Email is required', role: appError.role })
+    if (email === '') setError({ ...appError, email: 'Email is required' })
```
Comment (nidhigarg-bmw) [inline src/components/cax-responsibilities.tsx:112]: 'Email is required'  message use translation.

Comment (smartdeepp) [reply src/components/cax-responsibilities.tsx]: updated. :+1:

**Review by nidhigarg-bmw — APPROVED** [review]
Okay now

**Pull request merged**